<ui version="4.0" >
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>272</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="gbFind" >
     <property name="title" >
      <string>Find</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lblPattern" >
        <property name="text" >
         <string>Pattern:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>cmbPattern</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KComboBox" name="cmbPattern" >
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="duplicatesEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblFolder" >
        <property name="text" >
         <string>Folder:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>cmbDir</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" name="_3" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="KUrlComboRequester" name="cmbDir" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSync" >
          <property name="toolTip" >
           <string>Use the current document's path.</string>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblFiles" >
        <property name="text" >
         <string>Filter:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>cmbFilter</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KComboBox" name="cmbFilter" >
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="duplicatesEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="gbOptions" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="chkCaseSensitive" >
        <property name="text" >
         <string>Case sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRegExp" >
        <property name="text" >
         <string>Regular expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRecursive" >
        <property name="text" >
         <string>Recursive search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
